frames/src/main/java/com/checkout/frames/style/theme/DefaultPaymentFormTheme.kt

```diff
@@ -81,6 +81,12 @@ public object DefaultPaymentFormTheme {
         .setPaymentFormField(PaymentFormComponentField.BillingDetailsHeaderButton)
         .build()
 
+    private val billingFormCardHolderName = PaymentFormComponentBuilder()
```

Comment:
Yes, it is used right away during initialisation because it provides a list of default components from the builder when merchants use the theming capabilities of the SDK